data/chameleoncloud/sites/uc/clusters/chameleon/nodes/99732d8f-3f6c-4602-985a-35aff2657b6c.json

```diff
@@ -134,4 +143,4 @@
   },
   "type": "node",
   "uid": "99732d8f-3f6c-4602-985a-35aff2657b6c"
-}
+}
```

Comment:
we should probably make sure the tool adds a newline at the end of all the json files, otherwise it makes some editors sad.